.vscode/settings.json

```diff
@@ -10,5 +10,14 @@
         "**/node_modules": true,
         "dist/": true
     },
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "editor.formatOnSave": true,
+    "prettier.singleQuote": false,
+    "prettier.trailingComma": "es5",
+    "prettier.typescriptEnable": [
+        "typescript",
+        "typescriptreact"
+    ],
+    "prettier.semi": false,
+    "prettier.printWidth": 120
```

Comment:
Personally: I'd rather have awkward interfaces than awkward code reviews, 80 columns is so hard to write JS to, function definitions get really awkward across many lines as we have the types which make them much longer